### PR TITLE
DYN-6012 Add new solution to the build order

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

- [x] Add a new build.xml for NET6 solution build
- [x] Update Nuget to v6.6.1.2 because msbuild will try to do this anyway if dev running the build locally
image

- [ ] Related PR on buildscript repo to append a msbuild step for NET6 solution https://git.autodesk.com/Dynamo/DynamoBuildscripts/pull/37

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

Support .Net6 build in CI

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/build.xml [hunk 1]
```diff
@@ -20,6 +24,7 @@
 <Target Name="RestorePackages">
   <Exec Condition="!Exists($(NuGetPath))" Command="powershell.exe -ExecutionPolicy ByPass -Command Invoke-WebRequest -Uri https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile $(NuGetPath)" />
   <Exec Command="$(NuGetPath) restore $(Solution)"/>
+  <Exec Command="$(NuGetPath) restore $(SolutionNET6)"/>
```
Comment (pinzart90) [inline src/build.xml:27]: We might need to set the target framework here

**Review by pinzart90 — APPROVED** [review]

**Comment by QilongTang** [comment]
@pinzart90 @sm6srw  If I comment out the regular solution build, the build passed and fail at a later stage
```
1) Invalid : E:\Builds\Dynamo_master\Dynamo\bin\AnyCPU\Release\ProtoTest.dll
File not found: E:\Builds\Dynamo_master\Dynamo\bin\AnyCPU\Release\ProtoTest.dll

2) Invalid : E:\Builds\Dynamo_master\Dynamo\bin\AnyCPU\Release\WorkflowTests.dll
File not found: E:\Builds\Dynamo_master\Dynamo\bin\AnyCPU\Release\WorkflowTests.dll
```
I think we cant build these two solution files back to back without some kind of clean up maybe. I can look at if we can insert those steps.

**Comment by pinzart90** [comment]
Did you try to run a clean before the net6 build?

**Comment by QilongTang** [comment]
> Did you try to run a clean before the net6 build?

What we do before is just specify `ProjectToBuild` so no chance to do clean up in between

**Pull request merged**